app/views/submissions/_filters.html.erb

```diff
@@ -51,5 +51,16 @@
       </div>
     </div>
 
+    <div class='row'>
+      <div class='col-xs-12 col-sm-12 col-md-12 col-lg-12'>
+        <div class='from-mlab'>
+          <span class='check-box-label'>
```

Comment:
Let's improve accessibility by changing this `<span>` to a `<label>` and giving it an attribute of `for='include_from_mlab'`. That will help screen readers associate the label text with the checkbox and allow users to click on the text to toggle the checkbox state.

With our current styles, that will also make the text bold. If that's fine with you, you can leave it at that. If you want the text to look the same as it currently does, you can apply some CSS somewhere for this specific element setting `font-weight: normal`.